.golangci.yml

```diff
@@ -69,6 +74,14 @@ linters-settings:
   gocritic:
     enabled-checks:
       - commentFormatting
+  forbidigo:
```

Comment:
With this it should be safe to remove [`tools/roxvet/analyzers/invalidoutputroxctl`](https://github.com/stackrox/stackrox/blob/b49ecfbdfb1924a9c55ae0c6675d79a7b457794f/tools/roxvet/analyzers/invalidoutputroxctl/analyzer.go) but it can be done as a follow up